mcu/atmega-hal/Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2018"
 [features]
 rt = ["avr-device/rt"]
 device-selected = []
+disable-unused-adc = []
```

Comment:
In general, features should always be opt-in instead of opt-out, because features are selected based on the _union_ of all features requested by downstream crates.  I.e. suppose crate A sets `disable-unused-adc` while crate B expects the pins to be available - this wouldn't compile.  If, OTOH you'd have an `additional-adc-pins` feature which crate A ignores and crate B sets explicitly - all is well.

This also means you can get rid of the `not()` you have to use everywhere.

Reply:
yep, this makes sense, thanks!